tests/tools/traffic-replay/Config.py

```diff
@@ -15,6 +17,15 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-These tools are meant to become it own repository in the furture. They are here at the moment to help accelerate progress at getting everything working.
+#SSL config
+ca_certs = "/home/persia/server.crt"
```

Comment:
This is what I had in my config file. Forgot to remove it. Nothing ``ego maniacal''